src/components/Link.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link as USWDSLink } from "@trussworks/react-uswds";
+import { Link as GatsbyLink } from "gatsby";
+
+// Pass only activeClassName and partiallyActive to GatsbyLink
+// Pass only fileType and size to file downloads
+const Link = ({
+  children,
+  to,
+  activeClassName,
+  partiallyActive,
+  variant,
+  fileType,
+  size,
+  ...other
+}) => {
+  // Assumes that any internal link will start with exactly one slash, and that anything else is external.
+  const internal = /^\/(?!\/)/.test(to);
+
+  // Assumed file download link structure
+  const file = /\.[0-9a-z]+$/i.test(to);
+
+  if (variant === "nav") {
+    // Navigation link
+    return (
+      // TODO: there's no special attribute/component that Gatsby has for navigation, so should Gatsby links be used here?
```

Comment:
Is a nav link functionally the same as a normal link, but stylistically different, as it is for react router dom links (https://reactrouter.com/web/api/NavLink)? If so, then I think we should use gatsby for internal nav links

Reply:
A nav link can be the same as a normal link with extra classes added. I use Gatsby for internal nav links and USWDS for external nav links in the most recent push.